packages/financial-templates-lib/package.json

```diff
@@ -4,14 +4,17 @@
   "description": "Arbitrage automation and libraries for UMA financial templates",
   "dependencies": {
     "@ethersproject/bignumber": "^5.0.5",
+    "@google-cloud/bigquery": "^5.3.0",
     "@google-cloud/logging-winston": "^3.0.6",
     "@google-cloud/trace-agent": "^4.2.5",
     "@uma/common": "^1.0.1",
     "@uma/core": "^1.1.0",
     "@uniswap/sdk": "^2.0.5",
     "bluebird": "^3.7.2",
     "dotenv": "^6.2.0",
+    "highland": "^2.13.5",
     "minimist": "^1.2.0",
+    "moment": "^2.29.1",
```

Comment:
`moment` is also imported in `core`, should we move this to the main package.json in `protocol`?